packages/matter-node.js/src/matter/interaction/InteractionServer.ts

```diff
@@ -45,22 +45,22 @@ export const INTERACTION_PROTOCOL_ID = 0x0001;
 
 const logger = Logger.get("InteractionProtocol");
 
-export class ClusterServer<F extends BitSchema, A extends Attributes, C extends Commands, E extends Events> {
+export class ClusterServer<F extends BitSchema, SF extends Partial<TypeFromBitSchema<F>>, A extends Attributes, C extends Commands, E extends Events> {
     readonly id: number;
     readonly name: string;
     readonly attributes = <AttributeServers<A>>{};
     readonly commands = new Array<CommandServer<any, any>>();
 
-    constructor(clusterDef: Cluster<F, A, C, E>, features: TypeFromBitSchema<F>, attributesInitialValues: AttributeInitialValues<A>, handlers: ClusterServerHandlers<Cluster<F, A, C, E>>) {
-        const { id, name, attributes: attributeDefs, commands: commandDefs } = clusterDef;
+    constructor(clusterDef: Cluster<F, SF, A, C, E>, features: Omit<TypeFromBitSchema<F>, keyof SF>, attributesInitialValues: AttributeInitialValues<A>, handlers: ClusterServerHandlers<Cluster<F, SF, A, C, E>>) {
```

Comment:
why we need "features" here at all in contructor? Because with the "feature specific cluster definitions" the only feature info comes via "supportedFeatres" out of the definition. right?